layers/meta-balena-5x-owa/recipes-core/images/balena-image.inc

```diff
@@ -9,7 +9,7 @@ IMAGE_CMD:balenaos-img:append () {
     dd if=${DEPLOY_DIR_IMAGE}/${MACHINE}-flash.bin of=${BALENA_RAW_IMG} conv=notrunc seek=32 bs=1K
 }
 
-BALENA_BOOT_SIZE = "50000"
```

Comment:
Are you using this OS in production yet? This is a breaking change that will not allow you to do host OS updates because this changes the disk layout

Reply:
No, we are not using it in host OS Updates, indeed we are trying to test the host OS update before deploying it in production.
In order to test it we need at least a couple of relase images on BalenCloud, one older than the other. We've been trying to get a new release for some time now, so let's keep momentum and solve these issues you've commented.
I'll try to split the changes and follow your instructions asap. 
Thanks for the quick response!

Reply:
The reason behind this boot partition change is: 
As we only have one balena-owa5x release image on BalenaCloud we coudn't test hostOS update, but yesterday I found a trick that might let do it. I just modified the /etc/os-release file on the device to fake downgrade it. Then BalenaCloud recognized its version as an older one a offer the update. 
After launching it just failed with this error: 

_Mar 21 15:12:01 f38cf9f d3060640d90f[2404]: [ERROR] Boot files copy operations will fail with out of space error._

Then I increased the boot partition and tested on a local baked image, which seemed to have fixed the error. 

is it okay?
Thanks!